crates/node_binding/src/dependencies/entry_dependency.rs

```diff
@@ -0,0 +1,66 @@
+use std::any::TypeId;
+
+use napi::{Either, Env, JsString};
+
+#[napi]
+#[repr(C)]
+pub struct EntryDependency {
+  #[allow(dead_code)]
+  type_id: TypeId,
```

Comment:
what happens if we forget to use repr(c) and add type_id for some dependency? can it be checked in compile time or will cause segfault during runtime?

Reply:
this is a UB. napi-rs currently has same problem. I will fix this problem in napi-rs.

https://github.com/napi-rs/napi-rs/issues/2484